src/google/protobuf/compiler/objectivec/objectivec_helpers.h

```diff
@@ -167,7 +167,15 @@ string GetOptionalDeprecatedAttribute(
     isDeprecated = file->options().deprecated();
   }
   if (isDeprecated) {
-    string result = "GPB_DEPRECATED";
+    string message = descriptor->full_name();
+    const FileDescriptor* sourceFile = descriptor->file();
+    if (sourceFile != nullptr) {
```

Comment:
skip this, just build the string, if a descriptor really didn't have a file, the whole thing could come crashing down.  We wouldn't know the package, syntax version, etc.

Reply:
Done.